device.yml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 ---
-# yaml-language-server: $schema=https://harp-tech.org/draft-02/schema/device.json
+# yaml-language-server: $schema=https://raw.githubusercontent.com/harp-tech/protocol/main/schema/device.json
```

Comment:
```suggestion
# yaml-language-server: $schema=https://harp-tech.org/draft-02/schema/device.json
```

This schema link was correct in my view, the official link to the schema is https://harp-tech.org/draft-02/schema/device.json which is also the `$id` inside the `device.json` file.

Basically this acts like a DOI where even if we migrate the files themselves into different folders, repositories, etc. the publishing link itself will always be in the same predictable location in harp-tech.org.